js/lib/beautify-html.js

```diff
@@ -108,6 +108,24 @@
 
         options = options || {};
 
+        // Allow the inclusion of css and js options for <style> and <script> blocks
+        // inside html files (this is useful for plugins like Sublime's HTML prettify)
+        if (!options.js) {
+          options.js = {};
+        }
+        var js_beautify = function(js_source_text, options) {
+            options = options || options.js;
```

Comment:
This is an interesting start, but what you really want is something that overlays any values in `options.js` onto `options` object. 

And don't forget to do the same with `options.html`.

So, for instance your the following settings should work and result in html files with 8 space indents, `<style>` blocks wth 4 space indent, and `<script>` blocks with 2-space indents. 

```json
{
    "indent_size": 5,
    js: {
        "indent_size": 2
    },
    html: {
        "indent_size": 8
    }
}
```